app/views/tagging_spreadsheets/_import_preview.html.erb

```diff
@@ -2,7 +2,9 @@
   <br/>
 
   <div data-module="import-progress">
-    <%= render partial: "import_progress_bar", locals: { tag_mappings: tag_mappings } %>
+    <% if tag_mappings.any? %>
```

Comment:
Yeah, good point, I'll do that in a follow up PR to address the workflow UI (making the state of the import clearer).